data/126/021/452/3/1260214523.geojson

```diff
@@ -6,21 +6,21 @@
     "edtf:inception":"uuuu",
     "geom:area":0.0,
     "geom:area_square_m":0.0,
-    "geom:bbox":"-10.2,11.73333,-10.2,11.73333",
+    "geom:bbox":"-10.18,11.73333,-10.18,11.73333",
     "geom:latitude":11.73333,
-    "geom:longitude":-10.2,
+    "geom:longitude":-10.18,
     "gn:admin1_code":"K",
     "gn:admin2_code":"07",
-    "gn:asciiname":"Tomba Dougoula",
+    "gn:asciiname":"Tomba kanssa",
     "gn:country_code":"GN",
     "gn:dem":403,
-    "gn:feature_class":"P",
-    "gn:feature_code":"PPL",
+    "gn:feature_class":"A",
+    "gn:feature_code":"ADM3",
     "gn:geonameid":2414641,
     "gn:latitude":11.73333,
     "gn:longitude":-10.2,
-    "gn:modification_date":"2013-08-06",
-    "gn:name":"Tomba Dougoula",
+    "gn:modification_date":"2023-12-07",
```

Comment:
WOF exportify tool will generate a date in unix timestamp format for you automatically, but puts it in `wof:modified`.